go/cmd/vtworker/vtworker.go

```diff
@@ -78,9 +78,11 @@ func main() {
 		go func() {
 			if err := wi.WaitForCommand(worker, done); err != nil {
 				log.Error(err)
+				log.Flush()
```

Comment:
Not sure that will import properly, I know google's internal log doesn't have that... logutil has something I think.